packages/kmod-6.1-nvidia/nvidia-fabricmanager.service

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=NVIDIA fabric manager service
+After=network-online.target
+Requires=network-online.target
```

Comment:
I'm not confident we actually need these dependencies. It doesn't seem like Fabric Manager ought to care whether `eth0` has an IP address.

Reply:
removed.